docs/content/docs/deployment/configuration.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Could the document of configuration be generated by `OperatorConfigOptions`?

Reply:
Not very easy to do unfortunately. flink-docs contains the annotaiton processing logic for flink but it's not flexible